packages/backend-for-frontend/src/services/selfcareService.ts

```diff
@@ -45,16 +47,17 @@ export function selfcareServiceBuilder(
         `Retrieving User with with istitution id ${institutionId}, user ${userId}`
       );
 
-      const users = await selfcareV2Client.getInstitutionProductUsersUsingGET({
```

Comment:
This method is also used in `authorization-process` in the `assertUserSelfcareSecurityPrivileges` validator, should it be replaced there as well?

Reply:
The `getInstitutionProductUsersUsingGET` (this is what is used in the assertion) method has not changed, this PR is to replace the now deprecated `getInstitutionsUsingGET` with `getUserInstitutionUsingGET`.
The changes that are present in PR are derived from the `lint` because I changed `selfcareV2Client` to `selfcareV2InstitutionClient`

Reply:
True, my bad, the GitHub diff tricked me :)

Reply:
😆